rule/contract.go

```diff
@@ -70,3 +72,22 @@ type Contract struct {
 type TypedExpression interface {
 	Contract() Contract
 }
+
+// GetOperatorExpr returns an Expr that matches the provided operator
+// name. If no matching expression exists, and error will be returned.
+func GetOperatorExpr(name string) (Expr, error) {
+	switch name {
+	case "eq":
+		return &exprEq{}, nil
+	case "not":
+		return &exprNot{}, nil
+	case "and":
+		return &exprAnd{}, nil
+	case "or":
+		return &exprOr{}, nil
+	case "in":
+		return &exprIn{}, nil
+
+	}
+	return nil, fmt.Errorf("No operator Expression called %q exists", name)
```

Comment:
🌷 🌷 Error strings should not be capitalized

Reply:
@YassineE agreed - and picking me up on not doing the right thing is a big part of what review is for.  I don't need flowers ever time you notice that I'm not good at these small things ;-)